js/src/logger.ts

```diff
@@ -1809,6 +1816,22 @@ function _urljoin(...parts: string[]): string {
   return parts.map((x) => x.replace(/^\//, "")).join("/");
 }
 
+function validateTags(tags: readonly string[]) {
+  if (!tags) {
```

Comment:
This should not be necessary (or use `tags.length`), because an empty array evaluates to true. But also I feel like it's just unnecessary short-circuiting.

Reply:
will just remove